src/casp/services/api/data_manager/cellarium_general.py

```diff
@@ -110,6 +110,20 @@ def get_model_by_name(self, model_name: str) -> models.CASModel:
 
             return model
 
+    def get_user_by_email(self, user_email: str) -> models.User:
+        """
+        Get user by email
+
+        :param user_email: Email of the user to get
+
+        :return: User object
+        """
+        with self.system_data_db_session_maker() as session:
+            user = session.query(models.User).filter(models.User.email == user_email).one_or_none()
```

Comment:
Wow, never saw `one_or_none` that seems handy!

Reply:
It is!